pkg/codegen/internal/test/testdata/output-funcs/nodejs-extras/tests/codegen.spec.ts

```diff
@@ -0,0 +1,182 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'mocha';
+import * as assert from 'assert';
```

Comment:
Nit: I don't feel too strongly about this, but we generally use double quotes in our other `ts` code. At some point we should make sure these files are linted (and style checked + auto formatted if/when we hook that up).